can/interfaces/ixxat/canlib.py

```diff
@@ -1,414 +1,20 @@
-    constants.CAN_INFO_STOP: "CAN stopped",
-    constants.CAN_INFO_RESET: "CAN reset",
-}
-
-CAN_ERROR_MESSAGES = {
-    constants.CAN_ERROR_STUFF: "CAN bit stuff error",
-    constants.CAN_ERROR_FORM: "CAN form error",
-    constants.CAN_ERROR_ACK: "CAN acknowledgment error",
-    constants.CAN_ERROR_BIT: "CAN bit error",
-    constants.CAN_ERROR_CRC: "CAN CRC error",
-    constants.CAN_ERROR_OTHER: "Other (unknown) CAN error",
-}
-
-CAN_STATUS_FLAGS = {
-    constants.CAN_STATUS_TXPEND: "transmission pending",
-    constants.CAN_STATUS_OVRRUN: "data overrun occurred",
-    constants.CAN_STATUS_ERRLIM: "error warning limit exceeded",
-    constants.CAN_STATUS_BUSOFF: "bus off",
-    constants.CAN_STATUS_ININIT: "init mode active",
-    constants.CAN_STATUS_BUSCERR: "bus coupling error",
-}
-# ----------------------------------------------------------------------------
+from typing import Optional
 
 
 class IXXATBus(BusABC):
     """The CAN Bus implemented for the IXXAT interface.
 
-    .. warning::
-
-        This interface does implement efficient filtering of messages, but
-        the filters have to be set in :meth:`~can.interfaces.ixxat.IXXATBus.__init__`
-        using the ``can_filters`` parameter. Using :meth:`~can.interfaces.ixxat.IXXATBus.set_filters`
-        does not work.
+    Based on the C implementation of IXXAT, two different dlls are provided by IXXAT, one to work with CAN,
+    the other with CAN-FD.
 
+    This class only delegates to related implementation (in calib_vcinpl or canlib_vcinpl2) class depending on fd user option.
     """
 
-    CHANNEL_BITRATES = {
-        0: {
-            10000: constants.CAN_BT0_10KB,
-            20000: constants.CAN_BT0_20KB,
-            50000: constants.CAN_BT0_50KB,
-            100000: constants.CAN_BT0_100KB,
-            125000: constants.CAN_BT0_125KB,
-            250000: constants.CAN_BT0_250KB,
-            500000: constants.CAN_BT0_500KB,
-            666000: constants.CAN_BT0_667KB,
-            666666: constants.CAN_BT0_667KB,
-            666667: constants.CAN_BT0_667KB,
-            667000: constants.CAN_BT0_667KB,
-            800000: constants.CAN_BT0_800KB,
-            1000000: constants.CAN_BT0_1000KB,
-        },
-        1: {
-            10000: constants.CAN_BT1_10KB,
-            20000: constants.CAN_BT1_20KB,
-            50000: constants.CAN_BT1_50KB,
-            100000: constants.CAN_BT1_100KB,
-            125000: constants.CAN_BT1_125KB,
-            250000: constants.CAN_BT1_250KB,
-            500000: constants.CAN_BT1_500KB,
-            666000: constants.CAN_BT1_667KB,
-            666666: constants.CAN_BT1_667KB,
-            666667: constants.CAN_BT1_667KB,
-            667000: constants.CAN_BT1_667KB,
-            800000: constants.CAN_BT1_800KB,
-            1000000: constants.CAN_BT1_1000KB,
-        },
-    }
-
-    def __init__(self, channel, can_filters=None, **kwargs):
+    def __init__(self, channel, can_filters=None, fd=False, **kwargs):
```

Comment:
Can you improve the signature and the docstring of this `__init__()` method? This one will be shown in the docs. Otherwise this looks fine :)

Reply:
Done.